src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -799,13 +799,15 @@ func (c controller) waitForOLMOperators() bool {
 			continue
 		}
 
-		err = c.ic.UpdateClusterOperator(context.TODO(), c.ClusterID, operator.Name, operatorStatus, csv.Status.Message)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to update olm %s status", operator.Name)
-			continue
-		}
+		if operator.Status != operatorStatus || (operator.StatusInfo != csv.Status.Message && csv.Status.Message != "") {
+			c.log.Infof("CSV %s is in status %s, message %s.", operator.Name, csv.Status.Phase, csv.Status.Message)
```

Comment:
Consider logging the change
```suggestion
			c.log.Infof("CSV %s updated, status: %s -> %s, message: %s -> %s.", operator.Name, operator.Status, csv.Status.Phase, operator.StatusInfo, csv.Status.Message)
```

Reply:
Good idea!